charts/mediawiki/templates/_helpers.tpl

```diff
@@ -112,6 +118,22 @@ Common deployment environment variables
 - name: MW_LOG_TO_STDERR
   value: "yes"
 {{- end }}
+- name: MW_SMTP_ENABLED
+{{- if .Values.mw.smtp.enabled }}
+  value: "yes"
+{{- else }}
+  value: "no"
+{{- end }}
+- name: MW_SMTP_HOST
```

Comment:
I wonder if these env var should also be wrapped up in a conditional some how?
But I don't think' it matters too much / we don't have a best practice right now

Reply:
Good question - it probably wouldnt change anything as far as I can see but it could be cleaner?